### PR TITLE
zolotarev commit

### PR DESCRIPTION
**Pull request by Anzo85**

### CODE REVIEW

**Comment by Anzo85** [comment]
My commit

File: atm/Zolotarev/zolotarev.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+q
```
Comment (parkhomenko) [inline atm/Zolotarev/zolotarev.txt:1]: Write a hello world program at least

Comment (parkhomenko) [reply atm/Zolotarev/zolotarev.txt:1]: rejected

**Review by parkhomenko — CHANGES_REQUESTED** [review]
rejected

**Pull request merged**